tf-modules/archive/replay_sqs_messages.tf

```diff
@@ -0,0 +1,95 @@
+data "aws_iam_policy_document" "replay_sqs_messages_policy" {
+
+  statement {
+    actions   = [
+      "s3:GetObject",
+      "s3:PutObject",
+      "s3:DeleteObject"
+    ]
+    resources = ["arn:aws:s3:::${var.system_bucket}/*"]
+  }
+
+  statement {
+    actions = [
+      "ec2:CreateNetworkInterface",
+      "ec2:DescribeNetworkInterfaces",
+      "ec2:DeleteNetworkInterface"
+    ]
+    resources = ["*"]
+  }
+
+  statement {
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:DescribeLogStreams",
+      "logs:PutLogEvents"
+    ]
+    resources = ["*"]
+  }
+  statement {
+    actions = [
+      "secretsmanager:GetSecretValue"
+    ]
+    resources = [var.rds_user_access_secret_arn]
+  }
```

Comment:
This is only necessary if the Lambda is talking to RDS directly, which I don't believe it is

Reply:
[`030ed97` (#2373)](https://github.com/nasa/cumulus/pull/2373/commits/030ed97f35a8892454a2f46e229a87b2bb447203#diff-7b93858158e237529239ab394d5f5c6428fd1d94d04af74213f3bad9f3606ea5L30)